librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -345,6 +345,44 @@ commands:
             summary: Summarize current block
             cname: cmd_disassemble_summarize_block
             args: []
+  - name: pF
+    summary: Print parsed ASN.1, PKCS, X590, DER, etc formats
+    subcommands:
+      - name: pFa
+        summary: Decode ASN.1 from current block
+        cname: cmd_print_asn1
+        args:
+          - name: len
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
+      - name: pFb
+        summary: Decode raw protobuf from current block
+        cname: cmd_print_protobuf
+        args:
+          - name: len
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
+      - name: pFp
+        summary: Decode PKCS7 from current block
+        cname: cmd_print_pkcs7
+        args:
+          - name: len
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
+      - name: pFx
+        summary: Decode X.509 from current block
+        cname: cmd_print_x509
+        args:
+          - name: len
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
+      - name: pFA
+        summary: Decode Android Binary XML from current block
+        cname: cmd_print_axml
+        args:
+          - name: len
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
```

Comment:
in the command you do not supply this value, but just use the `core->block[size]` variables. i guess we can remove the args.

Reply:
yes, same for all the others please